include/bluetooth/bluetooth.h

```diff
@@ -681,6 +681,11 @@ int bt_le_ext_adv_stop(struct bt_le_ext_adv *adv);
  *  currently advertising then the advertising data will be updated in
  *  subsequent advertising events.
  *
+ *  When extended advertising @ref BT_LE_ADV_OPT_EXT_ADV is enabled and
+ *  @ref BT_LE_ADV_OPT_SCANNABLE is enabled then advertising data is ignored.
+ *  When @ref BT_LE_ADV_OPT_SCANNABLE is not enabled then scan response data is
+ *  ignored.
+ *
```

Comment:
```suggestion
 *  When both @ref BT_LE_ADV_OPT_EXT_ADV and @ref BT_LE_ADV_OPT_SCANNABLE are enabled,
 *  then advertising data is ignored.
 *  When @ref BT_LE_ADV_OPT_SCANNABLE is not enabled, then scan response data is
 *  ignored.
 *
```

Reply:
Thanks, I disagree on the use of `,` though :)